src/graphql/models/Article.js

```diff
@@ -529,6 +530,21 @@ const Article = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Attachment hash to search or identify files',
     },
+    cooccurrences: {
+      type: new GraphQLList(Cooccurrence),
+      resolve: async ({ id }, args, { loaders }) =>
+        loaders.searchResultLoader.load({
+          index: 'cooccurrences',
+          type: 'doc',
+          body: {
+            query: {
+              match: {
```

Comment:
[Match query](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-match-query.html) is for analyzed fields.

For keyword fields such as `articleIds`, [analyzers are not needed](https://stackoverflow.com/questions/71841934/performance-of-match-vs-term-query-in-elasticsearch), thus we should use [term query](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-term-query.html) instead.

Note: using `match` here is probably why tests would pass without updating Elasticsearch index previously XD

Reply:
because it only works for "match", I used term query first, but failed. maybe need to try again.

Reply:
using term query now.